domain/src/test/java/jinsu/antilog/domain/StringOperandTest.kt

```diff
@@ -0,0 +1,31 @@
+package jinsu.antilog.domain
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Assert.assertThrows
+import org.junit.Test
+
+class StringOperandTest {
+
+    @Test
+    fun givenStringTypeNumber_whenStringOperandToDouble_thenTypeIsDouble() {
+        // given : 숫자 값 문자열이 주어지고
+        val operand = "5"
+        // when : StringOperand 를 Double 로 변환했을 때
+        // then : Double 값이 나온다.
+        assertThat(StringOperand(operand).toDouble())
+            .isEqualTo(5.0)
```

Comment:
테스트가 짧더라도 when / then을 나눠보는것은 어떨까요?

```suggestion
        // when : StringOperand 를 Double 로 변환했을 때
        val actualResult = StringOperand(operand).toDouble()
        
        // then : Double 값이 나온다.
        assertThat(actualResult).isEqualTo(5.0)
```